src/Sylius/Bundle/CoreBundle/spec/Form/EventSubscriber/CustomerRegistrationFormSubscriberSpec.php

```diff
@@ -62,16 +62,18 @@ function it_sets_user_for_existing_customer(
         $this->preSubmit($event);
     }
 
-    function it_throws_unexpected_type_exception_if_data_is_not_customer_type(
+    function it_does_nothing_if_data_is_not_customer_type(
         FormEvent $event,
         FormInterface $form,
         ShopUserInterface $user,
+        RepositoryInterface $customerRepository,
```

Comment:
As this is an argument of subscriber's constructor, it should be added as the first one here